### DISCUSSION TITLE
Cant check out the cart

### DISCUSSION THREAD

**Discussion by 1e0n-xx** [chr233/ASFEnhance#246] [提问求助 / Support]
when checking out the cart, by "https://checkout.steampowered.com/checkout/inittransaction/"
can't get the transid, instead of 

>  {'success': 2, 'purchaseresultdetail': 0, 'paymentmethod': 0, 'palpaltoken': None, 'transid': None}

------------------------------------------------------------------------------------------------------------------------
maybe steam updates the purchase security, when checkout needs the second login to verify the info.

**Comment by chr233** [comment]
what the plugin's version and ASF version do you use

**Comment by 1e0n-xx** [comment]
latest

**Comment by 1e0n-xx** [comment]
i try to call this api , but always face the verify page. 

> https://login.steampowered.com/jwt/finalizelogin

I searched it on the internet, seems there is no solution to solve it. 
what can I do, if I fail to init transaction

Thanks for your reply

**Comment by woctezuma** [comment]
What are you trying to purchase?

**Comment by chr233** [comment]
you'd login first, steam require fresh session before purchase
